src/Microsoft.Identity.Client/AppConfig/ConfidentialClientApplicationBuilder.cs

```diff
@@ -43,9 +43,12 @@ internal ConfidentialClientApplicationBuilder(ApplicationConfiguration configura
         }
 
         /// <summary>
+        /// Constructor of a ConfidentialClientApplicationBuilder from application configuration options
+        /// . See https://aka.ms/msal-net-application-configuration
```

Comment:
> . [](start = 11, length = 2)

nit: shouldn't the . be in the previous line?

Reply:
Addressed. Thanks!